cognito-post-authentication/src/main/java/no/unit/nva/cognito/service/UserApi.java

```diff
@@ -7,5 +7,5 @@
 
     Optional<User> getUser(String username);
 
-    void createUser(User user);
+    Optional<User> createUser(User user);
```

Comment:
Does an empty Optional mean failure? If yes, then failure should be indicated with an Exception. If not, what is the step after the user has not been created? You do not use the Optional in your production code. only in tests